### DISCUSSION TITLE
Online only meetings

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1193] [Ideas]
In our district I have a few groups moving to online only. I am only able to save their listing if I include a location address and mark it as unable to attend in person. This seems to be confusing people as even though the address is crossed out they see it as the meeting location. I add Online only to notes but am being asked to remove the address. Is this something you are thinking about as meetings are still transitioning?

Thank you,
Rose L

**Comment by kiyote33** [comment]
Hi @district6wc Rose,
We provide instructions on our admin meeting page for how meeting list online meetings should be entered. Have you tried following the suggested steps to use only an approximate address like your city, state, country for the Address field and something like your_city Online for the location field? Here's the afore mentioned instructions: 
![image](https://github.com/code4recovery/12-step-meeting-list/assets/57125427/691a6f3d-4c56-4e1a-bc47-e438a1b9a775)
Note the bottom line which applies to online-only meetings.

**Comment by district6wc** [comment]
Hi Wayne

When I try just the city, I have to have it in all 3 inputs or I cannot
save the page.
So it lists with Raleigh 3 xs, if I leave any of them black I cannot update
or publish

[image: image.png]


Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by joshreisner** [reply]
hi rose, email attachments don't make it into github, could you manually add that screenshot on the github thread? thanks

**Reply by district6wc** [reply]
![image](https://github.com/code4recovery/12-step-meeting-list/assets/1336342/daa25822-9520-4618-899c-d54207486444)
Wayne this is how it now appears

**Reply by joshreisner** [reply]
hi rose, it's possible to remove the top bolded one by removing the location name, and the bottom one by setting the region to empty

https://code4recovery.org/meetings/online-meeting-test

<img width="715" alt="Screenshot 2023-08-24 at 7 16 02 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/01ed8f92-9841-4dbb-aaa3-1246905af202">

**Comment by district6wc** [comment]
Hi Wayne

No I cannot update without all 3 inputs having content, let me try a dash
or something

Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Comment by district6wc** [comment]
That did not work it needs a location entered
Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Comment by joshreisner** [comment]
One other option to consider is to switch to the TSML UI appearance - it does not show location information when it's an approximate location. here 

TSML UI example: https://code4recovery.org/tsml-ui?meeting=online-meeting-test

Same meeting in legacy mode: https://code4recovery.org/meetings/online-meeting-test

**Comment by district6wc** [comment]
Wayne

Can I just simply switch the interface, do I need to download the meeting
database first?
Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by joshreisner** [reply]
you should be able to toggle to TSML UI and back at Dashboard > Meetings > Settings > User Interface Display without downloading

<img width="517" alt="Screenshot 2023-08-24 at 10 42 13 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/825fb689-45e1-4ea5-b452-980461eb07cf">

- josh
